docs/learn/learn-parathreads.md

```diff
@@ -12,7 +12,7 @@ Parathreads are an idea for parachains to temporarily participate (on a block by
 dedicated parachain slot. This is done through economically sharing the scarce resource of a
 _parachain slot_ among several competing resources (parathreads). Chains that otherwise would not be
 able to acquire a full parachain slot or do not find it economically sensible to do so, are enabled
-to participate in {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} shared security
+to participate in {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s shared security
```

Comment:
I move the apostrophes outside the brackets when possible just because it breaks syntax highlighting in vscode if placed inside.  It renders correctly either way though.